### DISCUSSION TITLE
Setting upload filesize to > 2MB in docker environment

### DISCUSSION THREAD

**Discussion by arminus** [Leantime/leantime#952] [Q&A] [answered]
I'm running Leantime with docker compose on a Raspberry Pi. Is there a way to configure post_max_size and upload_max_filesize in php.ini for the image or do I need to build my own image and override php.ini?

The same question basically applies for other config vars as listed in https://docs.leantime.io/#/installation/configuration - what's the recommended way to set these in a docker environment?

**Comment by marcelfolaron** [comment, marked as answer]
We can add that value into our config but that will require a new image. The other option I could see is to mount a volume containing the custom ini. That way you have full control.  The path in the docker image is `/usr/local/etc/php/conf.d/custom.ini`

**Reply by arminus** [reply]
Great, hadn't thought about the volume override option. Thanks for pointing that out!

**Comment by marcelfolaron** [comment]
Let me know if that works... I haven't tried it myself.
I assume you need to restart the instance every time you make a change.

**Comment by arminus** [comment]
Just tested this, it's working ok:

`docker cp leantime:/usr/local/etc/php/conf.d/custom.ini .`

added
```
post_max_size = 20M
upload_max_filesize = 20M
```

added this line to the docker-compose.yml volumes section:
`- ./custom.ini:/usr/local/etc/php/conf.d/custom.ini`

restarted the container
